docs/overview.md

```diff
@@ -1,19 +1,19 @@
 # Kubernetes User Documentation
 
-The Kubernetes API currently manages 3 main resources:
-* [pods](pods.md)
-* [replication controllers](replication-controller.md)
-* [services](services.md)
+Kubernetes is an open source orchestration system for containers (currently Docker containers). It schedules containers onto nodes in a compute cluster and actively manages these containers to ensure that the state of the cluster continually matches the user's intentions.
```

Comment:
Really it is the scaling (and thus load balancing) and deployment patterns (co-located containers+volumes with shared fate, rolling updates, etc.) that Kubernetes is designed to address. Maybe running in production more generally. 

We also got feedback at the contributor gathering that we should state the longer-term vision somewhere. 

For instance, while we're targeting stateless and cloud-friendly stateful services now, we eventually plan to support all kinds of workloads, including databases, batch workloads, stream processing, etc.

We should also state somewhere that it's a goal of Kubernetes to enable users to think about and manage their applications, and Kubernetes will take care of the infrastructure details (this was discussed #2003).